client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html

```diff
@@ -78,7 +78,17 @@ <h2>{{ 'Meetings' | translate }}</h2>
                 </div>
 
                 @if (meeting.location || meeting.start_time || meeting.end_time) {
-                    <div class="subtitle ellipsis-overflow">
+                    <div
+                        class="subtitle ellipsis-overflow"
+                        [ngClass]="{
+                            'align-w-title-w-one-icon': xor(
+                                xor(meeting.isTemplate, meeting.locked_from_inside),
+                                meeting.isArchived
+                            ),
+                            'align-w-title-w-two-icon':
+                                meeting.isArchived && (meeting.isTemplate || meeting.locked_from_inside)
+                        }"
+                    >
```

Comment:
```suggestion
                    @let numIcons = meeting.isArchived + meeting.isTemplate + meeting.locked_from_inside
                    <div
                        class="subtitle ellipsis-overflow"
                        [ngClass]="{
                            'align-w-title-w-one-icon': numIcons === 1,
                            'align-w-title-w-two-icon': numIcons >= 2
                        }"
                    >
```

And remove `xor` from `meeting-list.component.ts`

Reply:
meeting.locked_from_inside can be undefined which results in numIcons to be NaN